### PR TITLE
Feat html t1001

### PR DESCRIPTION
**Pull request by Abishek666**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Weather project</title>
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Ubuntu">
+
```
Comment (saikrishnanp) [inline src/index.html]: We can remove this and next empty lines which doesn't seems to be required

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        <link rel="stylesheet" type="text/css" href="index_style.css">
+
+    </head>
+    <body>
+        <div class="headerimg">
```
Comment (saikrishnanp) [inline src/index.html]: For class names lets stick with kebab casing, eg header-img.

For id names we'll prefer snake casing, eg headerImg

Comment (saikrishnanp) [reply src/index.html]: You can try to use this format for all the class and id names used

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <img alt="leftcloudimg" class="leftcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <img alt="rightcloudimg" class="rightcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <div class="topsegment">
+                <div id="topfront">
+                    <div id="topimg1">
+                        <img id="topimg" src="../Assets/HTML&CSS/Icons_for_cities/maseru.svg" alt="x">
+                    </div>
+
+                    <div id="cityselect" style="position: relative;">
```
Comment (saikrishnanp) [inline src/index.html]:
```suggestion
                    <div id="cityselect">
```
Let's avoid using inline styling. We should prefer only to use styles via class names or id names through separate stylesheet

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                                10:10: <small> 44 </small>
+                            </h6>
+                            <img alt="amimg" id="time-img" src="../Assets/HTML&CSS/GeneralImages&Icons/amState.svg">
+                        </div>
+                        <div id="date" class="a">
+                            <h6 id="date-text">01-MAR-2022</h6>
+                        </div>
+                    </div>
+                </div>
+                <div id="topmiddle">
```
Comment (saikrishnanp) [inline src/index.html]: Usage of ids can be minimized as id has more specificity than class. Also, id is unique and can't have the same names for two tags. 
Whereas class names help in reusing the css styles.

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,503 @@
+*{
+    margin: 0;
+    padding: 0;
```
Comment (saikrishnanp) [inline src/style.css]: Using these styles are fine, but you can also try looking into reset.css and normalise.css

This comment is not mandatory to be followed though

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+    box-sizing: border-box;
+    font-family: Ubuntu,sans-serif;
+    overscroll-behavior: none;
+}
+
+#rightdown
+{position:relative;
```
Comment (saikrishnanp) [inline src/style.css]:
```suggestion
#rightdown {
position:relative; 
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,503 @@
...
+}
+
+#celcius
+{
+    position: relative;
+}
+
+.headerimg{
+    background:linear-gradient(0deg, rgba(0 0 0 / 100%), rgba(0 0 0 / 30%)), url("../Assets/HTML&CSS/GeneralImages&Icons/background.svg");
+    color: beige;
```
Comment (saikrishnanp) [inline src/style.css]: Using color code is a good practise than using the color's name

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,503 @@
...
+    animation-duration: 40s;
+    animation-delay: -5;
+    animation-iteration-count:infinite;
+     position: absolute;
+      width: 30vw; 
+      opacity: 0.3;
+       height: 15vw;
+}
+
+@keyframes animcloudleft {
```
Comment (saikrishnanp) [inline src/style.css]: All the keyframes can be placed near to each other and can be placed at either the bottom or top for better readability

**Review by saikrishnanp — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (Abishek666) [reply src/index.html]: Sai, we are instructed to use kebab case for both id and class names.

File: src/index.html [hunk 9]
```diff
@@ -5,32 +5,27 @@
...
+        <div class="header-img">
+            <img alt="leftcloudimg" class="left-cloud" src="../Assets/HTML&CSS/GeneralImages&Icons/cloud-8121.png">
+            <img alt="rightcloudimg" class="right-cloud" src="../Assets/HTML&CSS/GeneralImages&Icons/cloud-8121.png">
+            <div class="top-segment">
+                <div class="top-front">
+                    <div class="top-img1">
+                        <img id="top-img" src="../Assets/HTML&CSS/Icons_for_cities/maseru.svg" alt="x">
+                    </div>
+                    <div class="city-select">
+                        <select id="option-select">
```
Comment (saikrishnanp) [inline src/index.html]: This dropdown only allows to select between the options and doesn't allow to type for filtering the options. Can you explore the relevant tag for this and update, which also allow us to type for any keywords and select one from the filtered options?

File: src/style.css (hunk 5 above)
Comment (Abishek666) [reply src/style.css]: Sure Sai, hereafter I will try to follow this

**Review by saikrishnanp — APPROVED** [review]

**Pull request merged**